include/idf/Client.hh

```diff
@@ -172,6 +195,9 @@ class Client : public Manageable {
 
     private:
 
+    /** Number of times to retry establishing the initial connection */
+    int retryLimit;
```

Comment:
This is generating a new compiler warning on macOs

> IDF/include/idf/Client.hh:175:9: warning: field 'serverPort' will be initialized after field 'retryLimit' [-Wreorder-ctor]
        serverPort(port),
        ^
IDF/source/idf/RemoteSpaceNavigator.cpp:23:5: note: in instantiation of member function 'idf::Client<idf::SpaceNavigator, idf::SpaceNavigatorCommands>::Client' requested here
    Client<SpaceNavigator, SpaceNavigatorCommands>(spaceNavigator, hostName, hostPort) {}
    ^
1 warning generated.

Moving this declaration below `serverPort` should fix